docs/cloud.md

```diff
@@ -0,0 +1,245 @@
+have inbound and outbound connectivity via an Internet Gateway. The worker
+nodes will only be on the private subnet.
+
+
+:::tip
+Unless you are running `eksctl` and `kubectl` from a machine which is in the
+same VPC and subnet as the EKS control plane, you will want to set
+`publicAccess: true`. If so, we strongly recommend using the
+`publicAccessCIDRs` setting to limit IP ranges which are allowed to interact
+with the EKS control plane. We recommend customers set this to a well-scoped
+network range as they see fit to restrict access to one or more operators in your
+corporate network.
+
+See the [eksctl cluster access](https://eksctl.io/usage/vpc-cluster-access/)
+documentation to learn more.
+:::
+
+:::tip
+`eksctl` allows additional cluster setup and access configuration beyond what
+we've shown here. Check out the [configuration file
+schema](https://eksctl.io/usage/schema/) to learn more.
+:::
+
+To verify that the EKS cluster is running, run:
+
+```
+kubectl get nodes
+```
+
+You should see a three node cluster, all with the status `Ready`.
+
+## Cluster ingress
+
+To allow external access to Feldera and other services running inside the EKS
+cluster, we recommend setting up a Kubernetes Ingress controller. This is
+useful to allow access to the Feldera Web Console or to issue REST API calls
+directly. 
+
+:::tip
+When you install an ingress controller, it uses an AWS load balancer to expose
+an IP address via a generated DNS name for external access. By default,
+creating ingress controllers leads to both an internal and external load
+balancer being created, with the external one being prefered to expose
+services. Most Feldera Cloud users should need to expose Feldera services only
+over an internal load balancer.
+:::
+
+To install an ingress controller where the external load balancer is disabled,
+first create the following file:
+
+```
+# File name: ingress-controller-config.yaml
+
+controller:
+  ingressClassByName: true
+
+  ingressClassResource:
+    name: nginx
+    enabled: true
+    default: false
+    controllerValue: "k8s.io/ingress-nginx-internal"
+
+  service:
+    # Disable the external LB
+    external:
+      enabled: false
+
+    # Enable the internal LB.
+    internal:
+      enabled: true
+      annotations:
+        service.beta.kubernetes.io/aws-load-balancer-internal: "true"
+        service.beta.kubernetes.io/aws-load-balancer-backend-protocol: tcp
+        service.beta.kubernetes.io/aws-load-balancer-cross-zone-load-balancing-enabled: 'true'
+        service.beta.kubernetes.io/aws-load-balancer-type: nlb
+```
+
+Then use Helm to install the nginx Ingress controller:
+
+```
+kubectl create ns ingress-nginx-internal
+helm repo add ingress-nginx https://kubernetes.github.io/ingress-nginx
+helm repo update
+helm upgrade --install ingress-nginx-internal ingress-nginx/ingress-nginx \
+  -n ingress-nginx-internal \
+  -f ingress-nginx.yaml
```

Comment:
In the rest of the text `ingress-controller-config.yaml` (the file we created above) is not mentioned in any of the commands, or is it a default filename that is searched for? Or should the -f argument be that file?

Reply:
Good catch. It is the file referenced by -f, so the file created earlier should be ingress-nginx.yaml.